pkg/runctl/run_manager.go

```diff
@@ -35,6 +37,12 @@ const (
 	tektonTaskRunName = "steward-jenkinsfile-runner"
 )
 
+var (
+	stripJenkinsAnnotationsFunc = k8s.StripAnnotationsFunc("jenkins")
+	stripJenkinsLabelsFunc      = k8s.StripLabelsFunc("jenkins")
+	stripTektonAnnotationsFunc  = k8s.StripAnnotationsFunc("tekton")
+)
```

Comment:
I don't like to have this as package-scoped variables. Please inline them as before and see my [previous comment](https://github.com/SAP/stewardci-core/pull/8#discussion_r339475824).